src/CMakeLists.txt

```diff
@@ -102,6 +114,9 @@ endif (MOAB_FOUND)
 set_source_files_properties(decay.cpp PROPERTIES COMPILE_FLAGS 
     "-O0")
 
+add_executable(alphad ${PROJECT_SOURCE_DIR}/src/ensdf_processing/ALPHAD/alphad.f)
+target_link_libraries(alphad pyne)
+
```

Comment:
@scopatz The executable is now built to pyne/build/src(although I'm not sure if its copied over to the install directory), however it is not properly linked.  When I attempt to use it, it complains that it cant find they symbols for methods in the nsdflib.f file it references.

It would appear to me that ```target_link_libraries(alphad pyne)``` is not linking nsdflib.f to the alphad executable?